CPF-two.md

```diff
@@ -0,0 +1,119 @@
+(apologies for cross-posting) 
+
+----------------------------------------------------------------
+
+*Workshop for NLP Open Source Software (NLP-OSS)*
+
+11 or 12 Nov 2020, Co-located with EMNLP 2020
+
+https://nlposs.github.io/2020
+
+Deadline for Long and Short Paper submission: 05 August 2020 
+
+(23:59, GMT-11)
+
+----------------------------------------------------------------
+
+You have used NLP open source tools and bore grievances but found the solution after hours of coffee and computer staring. Share that at NLP-OSS and suggest how open source could change for the better (e.g. best practices, documentation, API design etc.)
+
+You are tired of Zoom sessions at ACL but got inspired to write an open source for a nifty idea you got from ACL 2020. Share what you've built at NLP-OSS and what motivated you other than distracting yourself from attending Zoom QnAs; understanding about motivations of OSS forms a larger understanding of OSS sustainability in general.
+
+You came across an awesome SOTA system on NLP task X that once ruled the F1 score. But now the code is stale and it takes an dinosaur to understand the code. Share your experience at NLP-OSS and propose how to "replicate" these forgotten systems. 
+
+You read an NMT paper with SOTA BLEU scores. But now the code is stale and it to understand the code and difficult to reproduce the results. Share your experience at NLP-OSS and propose how to "replicate" these forgotten systems. 
```

Comment:
This looks very similar to the above sentence. Is this intentional?

Reply:
Ah this is unintended! let me try to combine them.

Reply:
First one seems more generic, I've the 2nd.